core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -2042,7 +2042,8 @@ class KafkaApis(val requestChannel: RequestChannel,
     authorizeClusterAction(request)
 
     val lastOffsetForLeaderEpoch = replicaManager.lastOffsetForLeaderEpoch(requestInfo.asScala).asJava
-    sendResponseExemptThrottle(request, new OffsetsForLeaderEpochResponse(lastOffsetForLeaderEpoch))
+    sendResponseExemptThrottle(request, new OffsetsForLeaderEpochResponse(AbstractResponse.DEFAULT_THROTTLE_TIME,
```

Comment:
nits: For other responses we have a constructor (e.g. `ListOffsetResponse`) that does not require throttleTime. Could we do the same for `OffsetsForLeaderEpochResponse` for consistency?